devcontainer/features/features.go

```diff
@@ -205,6 +205,7 @@ func (s *Spec) Compile(options map[string]any) (string, error) {
 	if comment != "" {
 		lines = append(lines, comment)
 	}
+	lines = append(lines, fmt.Sprintf("WORKDIR %s", s.Directory))
```

Comment:
nit:
```suggestion
	lines = append(lines, "WORKDIR "+s.Directory)
```
...is slightly simpler